jobrunner/etc/opensafely/ssh-banner

```diff
@@ -0,0 +1,7 @@
+This the OpenSAFELY TPP backend.
```

Comment:
as this is a TPP-specific message, should the filename be TPP-specific?

Reply:
Hmm, yeah not sure about this

Reply:
removed